edx/frontend-app-course-authoring
# feat: if configured for a provider disable configuring unless user is an admin [BD-38] [TNL-8442]

This PR changes the LTI Config form to hide the LTI configuration for providers that have the adminOnlyConfig flag set. When that flag is set the LTI configuration is no longer visible, or editable by users that aren't global admins. 

**JIRA tickets**: TNL-8442

**Dependencies**: https://github.com/edx/edx-platform/pull/28347

**Screenshots**: 

![Screenshot 2021-08-03 at 07-28-28 Course Authoring edX](https://user-images.githubusercontent.com/118837/127975845-0bdb9922-926b-4813-80f7-fe80d9ab467c.png)
Accessing YellowDig config as a global admin

![Screenshot 2021-08-03 at 07-31-40 Course Authoring edX](https://user-images.githubusercontent.com/118837/127977180-be3762d8-cc2d-4840-8699-60ea8a2d6ad8.png)
Accessing YellowDig config as course staff

**Merge deadline**: "None"

**Testing instructions**:

1. Set up the corresponding edx-platform PR: https://github.com/edx/edx-platform/pull/28347
2. Set up YellowDig config for a course as global admin
3. Try accessing YelloDig config as course staff, you should see a message, and not be able to configure the app
4. Try configuring another LTI provider like Piazza as both course staff and global staff. It should be accessible by both unlike YellowDig. 

**Reviewers**
